CHANGELOG.md

```diff
@@ -686,8 +688,9 @@ peripheral.register.write(|w| w.field().set());
 
 - Initial version of the `svd2rust` tool
 
-[Unreleased]: https://github.com/rust-embedded/svd2rust/compare/v0.22.2...HEAD
-[v0.22.2]: https://github.com/rust-embedded/svd2rust/compare/v0.22.1...v0.22.2
+[Unreleased]: https://github.com/rust-embedded/svd2rust/compare/v0.23.0...HEAD
+[v0.23.0]: https://github.com/rust-embedded/svd2rust/compare/v0.22.2...v0.23.0
+[v0.22.1]: https://github.com/rust-embedded/svd2rust/compare/v0.22.0...v0.22.1
```

Comment:
Seems like you accidentally removed v0.22.2 and replaced it by another v0.22.1 anchor

Reply:
fixed